.github/workflows/CI.yml

```
@@ -432,7 +433,7 @@ jobs:
         id: versioning
         run: |
           # build a version with github run id so that we can distingush each build for integ-test
-          Version="`cat VERSION`-$GITHUB_RUN_ID"
+          Version="$(cat VERSION)-$(git rev-parse --short HEAD)"
```

```suggestion
          version="$(cat VERSION)-$(git rev-parse --short HEAD)"
```
nitpick: Local variable names should be lower snake case. https://google.github.io/styleguide/shellguide.html#naming-conventions

---

The version output is using version but either way, I'm good with the nitpick lol